### ISSUE TITLE
Banner in my kanban board

### ISSUE THREAD

**Issue by dlion** [noatpad/obsidian-banners#20]
Hi, I'm using the [kanban plugin](http://matthewmeye.rs/obsidian-kanban/) and I would like to set a banner also on that note.
Apparently right now is not possible to visualize it without going to the "open as markdown" view as you can see from my screenshot:

![image](https://user-images.githubusercontent.com/2125236/144403140-ba177cd5-6f40-4419-abe2-04d02143fde6.png)

and this is the markdown view:

![image](https://user-images.githubusercontent.com/2125236/144403200-2edfd42b-c732-41b2-9cc6-920cc4f4e671.png)

Apparently, the DOM structure for the kanban view is quite different from the markdown one and it leads to do not having the banner appearing on the page. 

Thank you

**Comment by noatpad** [comment]
Yeah it's a whole new view entirely, so it probably needs a bit of special treatment. But I think it's totally doable as long as I can find a listener to look into, so I'll give it a look-see when I get the chance

**Comment by dlion** [comment]
Great, thanks!

**Comment by Oblique82** [comment]
Any progress on this? I am keen too
